cetmix_tower_webhook/readme/DESCRIPTION.md

```
@@ -0,0 +1,5 @@
+This module implements incoming webhooks for [Cetmix Tower](https://tower.cetmix.com). Webhooks are authorised using customisable authenticators which can be pre-configured and reused across multiple webhooks. Webhooks and authenticators can be exported and imported using YAML format, which makes them easily sharable.
+
+This module is a part of Cetmix Tower, however it can be used to manage any other odoo applications. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)